api/src/main/java/org/apache/iceberg/expressions/Predicate.java

```diff
@@ -62,10 +95,10 @@ public String toString() {
         return String.valueOf(ref()) + " == " + literal();
       case NOT_EQ:
         return String.valueOf(ref()) + " != " + literal();
-//      case IN:
-//        break;
-//      case NOT_IN:
-//        break;
+      case IN:
+        return String.valueOf(ref()) + " in " + literalSet();
```

Comment:
I think this should add parentheses around the set.

Reply:
Good point. Will update it accordingly. Thanks.